spec/support/feature_helpers.rb

```diff
@@ -70,8 +70,8 @@ def create_content_change(overrides = {})
     expect(response.status).to eq(202)
   end
 
-  def send_status_update(reference, status, expected_status: 204)
-    params = { reference: reference, status: status }
+  def send_status_update(sent_at, completed_at, reference, status, expected_status: 204)
```

Comment:
Similar to above, I would move these arguments after the `reference` and `status` arguments.